server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -13,6 +13,7 @@ CobolVisitor.progIDIssueMsg=There is an issue with PROGRAM-ID paragraph
 CobolVisitor.subroutineNotFound=%s: Subroutine not found
 semantics.redefinedContainValue=The redefining item cannot contain a VALUE clause: %s
 semantics.redefineImmediatelyFollow=REDEFINES line must immediately follow redefined item: %s
+semantics.redefineSameGroup=The redefining and redefined items must belong to the same group: %s
```

Comment:
I believe this should be re-phrased, lets ask to tech-writer
@zeibura

Reply:
@asatklichov this phrase was picked from Zeb's comment to the previous PR
https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/868#discussion_r613081745